### ISSUE TITLE
ID 256053: [Programmatic Access - Simon - Delete]: Visual label and aria label is not same for 'Yes' button under "delete' pop up.

### ISSUE THREAD

**Issue by eledobleefe** [grafana/grafana#66046]
### **User Experience:** 
Visually impaired users who depend on screen readers will get disoriented and will face difficulty in understanding the control if correct `aria-label` will not be defined for the controls. 
_Note: User credentials should NOT be included in the bug._

### **Repro Steps:**

1. Open #Url: [play.grafana.org](https://play.grafana.org/). 
2. Sign in with valid credentials. "Home" page appears. 
3. Navigate the Feature Showcase pane. Select any one of the data such as ,"Time Series Graphs". 
4. "Visualizations" page opens. Navigate to the "Soft Min Max" pane and expand it. 
5. Select the "Delete" button. Navigate the "Delete Row" pop-up. 
6. Turn on narrator. Verify the issue.

**Actual Result:** 
Issue: Visual label and `aria-label` is not same for 'Yes' button under "delete' pop up. 

**Observation:** 
Screen reader is reading 'Confirm modal danger button' when `focus` moves on yes button.

**Expected Result:**
Solution: Visual label and `aria-label` should be same as 'Yes' for 'Yes' button under "delete' pop up. Example: Screen reader should read as 'Yes' button.

_Note: The same issue exists with NVDA and JAWS. Same issue appears for Library Panels->Create New Panel->Save folder Same issue exists with Feature showcases->edit->dashboard list->Library Panels->filtered data->Replace with library panel->Replace._

### **Severity:**
`SEV2`

### **Area:**
`Confirm Modal` (Saga Design System)

**Comment by grafanabot** [comment]
@eledobleefe please add one or more appropriate labels. Here are some tips:

- if you are making an issue, TODO, or reminder for yourself or your team, please add one label that best describes the product or feature area. Please also add the issue to your project board. :rocket:

- if you are making an issue for any other reason (docs typo, you found a bug, etc), please add at least one label that best describes the product or feature that you are discussing (e.g. `area/alerting`, `datasource/loki`, `type/docs`, `type/bug`, etc). [Our issue triage](https://github.com/grafana/grafana/blob/main/ISSUE_TRIAGE.md#3-categorizing-an-issue) doc also provides additional guidance on labeling. :rocket:

 Thank you! :heart:

**Comment by ashharrison90** [comment]
@L-M-K-B i actually just fixed this yesterday accidentally with https://github.com/grafana/grafana/pull/67140 (see third bullet point in description

might be worth verifying locally, but i think we can just close this now 👍
